lib/parser/source/rewriter.rb

```diff
@@ -124,7 +140,8 @@ def insert_after(range, content)
       #
       def insert_after_multi(range, content)
         @insert_after_multi_order += 1
-        append Rewriter::Action.new(range.end, content, true, @insert_after_multi_order)
+        group_delta = range.empty? ? 0 : 0x1fff_ffff
+        append Rewriter::Action.new(range.end, content, true, @insert_after_multi_order - group_delta)
```

Comment:
Can we make this a bit more elegant? Pull `0x1fffffff` out and into a constant, at least.